constantTimeCompare.js

```diff
@@ -0,0 +1,46 @@
+  /**
+   * constantTimeCompare(x, y)
+   *
+   *  Created by Eric Elliott for the book,
```

Comment:
This is changing so much, I'll need to correct the book. Is it possible to issue book safety recalls, I wonder? ;)